pithos/pithos.py

```diff
@@ -415,6 +433,7 @@ def set_audio_quality(self, *ignore):
         self.worker_run('set_audio_quality', (self.settings.get_string('audio-quality'),))
 
     def pandora_connect(self, *ignore, message="Logging in...", callback=None):
+        self.connected = False
```

Comment:
Logically I think it makes more sense to have a `connected` property in the `Pandora` class if possible.